django/contrib/auth/forms.py

```diff
@@ -250,6 +250,7 @@ def save(self, domain_override=None,
                 'uid': urlsafe_base64_encode(force_bytes(user.pk)),
                 'user': user,
                 'token': token_generator.make_token(user),
+                'extra_email_context': extra_email_context,
```

Comment:
This test needs to cover the case where this line is inadvertently removed (i.e. should test that the extra context values can be used in the email template). Also, `extra_context` should be merged with `context`, e.g. `context.update(extra_email_context)`.